### DISCUSSION TITLE
Tree from BCR clone_id after BCR clustering

### DISCUSSION THREAD

**Discussion by saramoein372** [tuonglab/dandelion#108] [General]
Hi Kelvin,

I have a question: is there any function in dandelion that helps to visualize the results of BCR clustering? I have generated all clone_ids , and would like to visualize it. But I can not find any function in dandelion.

Do you have any comment for me? Thanks.

Best,
Sara

**Comment by zktuong** [comment]
Hi Sara,

the currently available visualization tools are described in:
https://sc-dandelion.readthedocs.io/en/latest/notebooks/4_dandelion_visualization-10x_data.html

Alternatively, you can look at https://icbi-lab.github.io/scirpy/latest/ and their range of plotting functions, which accepts data from dandelion.

You can also look up the immcantation suite for their range of visualization options.

If none of these are to your liking, you are welcomed to request for a specify type of visualization and I may look into it when I have the time, or you can try and come up with one yourself and put in a pull request later.
